clientSide/src/components/Product/Product.js

```diff
@@ -0,0 +1,33 @@
+import './Product.css';
+
+function Products(){
+    return(
+
+        <div className="color">
+            <div className="categories">
+                        <div className="design">
+                            <img src="https://i.etsystatic.com/26610308/r/il/723e68/2831141092/il_1588xN.2831141092_dr44.jpg" alt="" />
+                        </div>
+                        <div className="flower">
+                            <img src="https://img0.etsystatic.com/151/0/14285414/il_340x270.1156193620_ol78.jpg" alt="" />
+                        </div>
+                        <div className="hangingbells">
+                            <img src="resources/background1.png" alt="" />
+                        </div>
```

Comment:
Please create component for div and reuse it.

Reply:
i created please check it once